### PR TITLE
Refactored persistent drawer left

### PR DESCRIPTION
**Pull request by FireQueen-3010**
1) Changed background color
2) Refactored logo on the drawer
3) Refactored color or icons and text
4) Added slight hover effects on the links

Desktop view:
image

Mobile view:
image

### CODE REVIEW

**Comment by FireQueen-3010** [comment]
Please review @ArunTeltia

File: src/components/PersistentDrawerLeft/PersistentDrawerLeft.css [hunk 1]
```diff
@@ -43,3 +43,19 @@
...
+.drawer_sakura{
+  color: #56cc9d;
+}
+
+.drawer_project{
+  color:whitesmoke;
+}
+
+.drawer_elements:hover{
+  background-color: #2c2b2b8c !important;
```
[the remark this answers is not in the document]
Comment (FireQueen-3010) [reply src/components/PersistentDrawerLeft/PersistentDrawerLeft.css:60]: Actually without writing the !important, the code didn't work but I will try to move it to the last line and see if it works :)

**Comment by LordShenron** [comment]
There is fundamental issues in this design..
1. Why are the topnames clickable if we are not using them as sub drawers. 
2. Why are the topnames look literally them same as other elements of the list. If we are using them as headings then they should look completely diff from normal list elements. 
3. Because of the 2nd issue i have already clicked the heading atleast a 100 times xD

Open a separate issues if you wish to work on these @FireQueen-3010

**Comment by LordShenron** [comment]
For an example: 
image
Here I instinctively know that "Talk with us" is the heading and other elements are clickable buttons.

**Comment by FireQueen-3010** [comment]
> There is fundamental issues in this design..
> 
> 1. Why are the topnames clickable if we are not using them as sub drawers.
> 2. Why are the topnames look literally them same as other elements of the list. If we are using them as headings then they should look completely diff from normal list elements.
> 3. Because of the 2nd issue i have already clicked the heading atleast a 100 times xD
> 
> Open a separate issues if you wish to work on these @FireQueen-3010

In the bread-crum cards? @LordShenron

**Comment by LordShenron** [comment]
> > There is fundamental issues in this design..
> > 
> > 1. Why are the topnames clickable if we are not using them as sub drawers.
> > 2. Why are the topnames look literally them same as other elements of the list. If we are using them as headings then they should look completely diff from normal list elements.
> > 3. Because of the 2nd issue i have already clicked the heading atleast a 100 times xD
> > 
> > Open a separate issues if you wish to work on these @FireQueen-3010
> 
> In the bread-crum cards? @LordShenron. I'll raise issues for them and work on them :)

In the sidenav

**Comment by FireQueen-3010** [comment]
> For an example:
> image
> Here I instinctively know that "Talk with us" is the heading and other elements are clickable buttons.

This one is the bread-crum card, do I raise another issue for this as well?

**Comment by LordShenron** [comment]
> > There is fundamental issues in this design..
> > 
> > 1. Why are the topnames clickable if we are not using them as sub drawers.
> > 2. Why are the topnames look literally them same as other elements of the list. If we are using them as headings then they should look completely diff from normal list elements.
> > 3. Because of the 2nd issue i have already clicked the heading atleast a 100 times xD
> > 
> > Open a separate issues if you wish to work on these @FireQueen-3010
> 
> This one I suppose is for side-nav, could you please give an example, I'm unable to follow actually @LordShenron

image
I actually tried to do it myself and This is how it should look.

**Comment by FireQueen-3010** [comment]
> > > There is fundamental issues in this design..
> > > 
> > > 1. Why are the topnames clickable if we are not using them as sub drawers.
> > > 2. Why are the topnames look literally them same as other elements of the list. If we are using them as headings then they should look completely diff from normal list elements.
> > > 3. Because of the 2nd issue i have already clicked the heading atleast a 100 times xD
> > > 
> > > Open a separate issues if you wish to work on these @FireQueen-3010
> > 
> > 
> > This one I suppose is for side-nav, could you please give an example, I'm unable to follow actually @LordShenron
> 
> image
> I actually tried to do it myself and This is how it should look.

Oh got it now, do I work on this or you completely done with designing it already ? @LordShenron

**Comment by FireQueen-3010** [comment]
> > For an example:
> > image
> > Here I instinctively know that "Talk with us" is the heading and other elements are clickable buttons.
> 
> This one is the bread-crum card, do I raise another issue for this as well? @LordShenron

And wb this? @LordShenron

**Comment by LordShenron** [comment]
> Oh got it now, do I work on this or you completely done with designing it already ? @LordShenron

I can push the changes till the point i have done it. But it still look weird coz the topname element have greyish color it should be white and Bold.

**Comment by LordShenron** [comment]
> And wb this? @LordShenron

That was never an issue.. I used it to tell how sidenav should look.

**Comment by FireQueen-3010** [comment]
> > Oh got it now, do I work on this or you completely done with designing it already ? @LordShenron
> 
> I can push the changes till the point i have done it. But it still look weird coz the topname element have greyish color it should be white and Bold.

Ok  I'll make the required changes after you push it :)

**Comment by FireQueen-3010** [comment]
> > And wb this? @LordShenron
> 
> That was never an issue.. I used it to tell how sidenav should look.

Ok, sry my bad :p

**Pull request merged**